src/client/tweetAnalysis.js

```diff
@@ -7,14 +34,14 @@
  */
 function countWords(tweets) {
   const wordCount = {};
-  for (let tweet of tweets) {
-    for (let word of tweet.content.split(/[ !,.?"']/)) {
-      word = word.trim();
-      if (word === '') { continue; }
-
-      // ~~ will convert floats to integer,
-      // but importantly quickly convert undefined to 0
-      wordCount[word] = ~~wordCount[word] + 1;

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2158438031/issues/source?bid=3148782&fileBranchId=3159641#l37)